dd-sdk-android-core/api/dd-sdk-android-core.api

```diff
@@ -77,6 +77,7 @@ public abstract interface class com/datadog/android/api/InternalLogger {
 	public static final field Companion Lcom/datadog/android/api/InternalLogger$Companion;
 	public abstract fun log (Lcom/datadog/android/api/InternalLogger$Level;Lcom/datadog/android/api/InternalLogger$Target;Lkotlin/jvm/functions/Function0;Ljava/lang/Throwable;ZLjava/util/Map;)V
 	public abstract fun log (Lcom/datadog/android/api/InternalLogger$Level;Ljava/util/List;Lkotlin/jvm/functions/Function0;Ljava/lang/Throwable;ZLjava/util/Map;)V
+	public abstract fun metric (Lkotlin/jvm/functions/Function0;Ljava/util/Map;)V
```

Comment:
this has to be updated

Reply:
yeah it is something wrong with the api generation as I already ran that method 🤔